dm/ctl/common/config.go

```diff
@@ -18,12 +18,17 @@ import (
 	"flag"
 	"fmt"
 	"net"
+	"time"
 
 	"github.com/BurntSushi/toml"
 	"github.com/pingcap/dm/pkg/utils"
 	"github.com/pingcap/errors"
 )
 
+var (
+	defaultRPCTimeout = "10m"
```

Comment:
is appropriate with `10m` in the code but `5s` in _dmctl.toml_?

Reply:
use `10m` unified